pkg/resource/aws/aws_route53_health_check_ext_test.go

```diff
@@ -0,0 +1,50 @@
+package aws
+
+import (
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+)
+
+func TestAwsRoute53HealthCheck_String(t *testing.T) {
+	tests := []struct {
+		name        string
+		healthCheck *AwsRoute53HealthCheck
+		want        string
+	}{
+		{
+			name:        "Just id",
+			healthCheck: &AwsRoute53HealthCheck{Id: "id"},
+			want:        "id",
+		},
+		{
+			name: "Tag name with fqdn and respath",
+			healthCheck: &AwsRoute53HealthCheck{
+				Id:           "id",
+				Tags:         map[string]string{"Name": "name"},
+				Fqdn:         aws.String("fq.dn"),
+				ResourcePath: aws.String("/toto"),
+			},
+			want: "name (fqdn: fq.dn path: /toto)",
```

Comment:
I think it will be more consistent with other resources (and more readable) if we separate fields with coma here, like `name (fqdn: fq.dn, path: /toto)`